framework/platform/bpl/include/bpl/bpl.h

```diff
@@ -9,9 +9,7 @@
 #ifndef _BPL_H_
 #define _BPL_H_
 
-#ifdef __cplusplus
-extern "C" {
-#endif
+namespace bpl {
```

Comment:
Since the namespace is such a big change, I would have done the removal of the `extern "C"` in a separate commit. But that's just a nitpick.

More importantly, though: I would take this occasion to also remove the `bpl_` prefix from all names. The namespace replaces it.

Also, did you make this change automatically (e.g. with a sed script)? If yes, please mention the command you used in the commit message, and explicitly mention which additional manual changes you made.

Reply:
I was going to make this very same proposal 
> More importantly, though: I would take this occasion to also remove the bpl_ prefix from all names. The namespace replaces it.

I didn't use any script to make the changes, all changes where made manually.